Pipfile

```diff
@@ -10,6 +10,7 @@ urllib3 = ">=1.23"
 python-dateutil = ">=2.8.1"
 pytz = "*"
 jsonschema = "==4.0.0"
+pyyaml = ">=5.4.1"
```

Comment:
todo: Please add this in the setup.py file as well.

Reply:
Its already added.